src/pages/Controllers/Controllers.js

```diff
@@ -90,15 +91,26 @@ function Details() {
     </span>
   );
 
+  function generatePathValue(c) {
+    const column = { content: "" };
+    column.content = c.path === "admin/jaas" ? "JAAS" : c.path;
+    if (!column.content) {
```

Comment:
I think it would be much more clear to use a set of 'if' statements here to set column.content. you are mixing ternary operator with traditional if statement to achieve this. The use of different techniques takes a min for the reader to understand it's all part of the same logic.

Reply:
I've updated this to use a single set of conditionals.